convert.go

```diff
@@ -0,0 +1,140 @@
+package godash
```

Comment:
Maybe we should put this file into a `string` subpackage, so it can be used as `godash.string.ToX(str)`?

Reply:
Sounds good! Should we also split the functions to their own files?

Reply:
Probably, that's the go way 😄